plugins/plugin-debugger/che-plugin-ext-debugger-client/src/main/java/org/eclipse/che/ide/ext/debugger/client/debug/DebuggerServiceClient.java

```diff
@@ -41,19 +46,24 @@
     Promise<Void> disconnect(@NotNull String id);
 
     /**
+     * Attach debugger.
+     */
+    Promise<DebuggerInfo> isConnected(@NotNull String id);
```

Comment:
Why it is named as isConnected? When I see isConnected I expect get boolean as result... Also java docs says about 'Attach' maybe it would be better to name it as 'attach'? WDYT?